### PR TITLE
Downgrade some logging to debug in fipreports

### PR DESCRIPTION
**Pull request by berland**
Solve pylint import issue

### CODE REVIEW

File: ecl2df/fipreports.py [hunk 1]
```diff
@@ -176,13 +176,16 @@ def fill_parser(parser):
         "-o", "--output", type=str, help="Output CSV filename", default="outflow.csv"
     )
     parser.add_argument("-v", "--verbose", action="store_true", help="Be verbose")
+    parser.add_argument("--debug", action="store_true", help="Be even more verbose")
```
Comment (lars-petter-hauge) [inline ecl2df/fipreports.py:179]: Be verbosest? Oh, the humor! :)

**Comment by lars-petter-hauge** [comment]
>Solve pylint import issue
Is this the change to explicit imports?

File: ecl2df/fipreports.py (hunk 1 above)
Comment (berland) [reply ecl2df/fipreports.py:179]: Changed to a duller `Debug mode for logging` ... :)

**Comment by berland** [comment]
Commit message changed on the import issue.

**Comment by lars-petter-hauge** [comment]
`avoid pyling´ -> `pylint`, depending on your level of correctness, I'll leave it up to you to decide.

**Review by lars-petter-hauge — APPROVED** [review]
LGTM

**Comment by berland** [comment]
> `avoid pyling´ -> `pylint`, depending on your level of correctness, I'll leave it up to you to decide.

Fixed in github squash ui.

**Pull request merged**